index.html

```diff
@@ -78,7 +79,7 @@ <h2>Promo video of the game</h2>
 		<div class="featured-bg set-bg" data-setbg="img/featured-bg.png"></div>
 		<div class="featured-box">
 			<div class="text-box">
-				<div class="top-meta">11.04.21  /  in <a href="game-single.html">Games</a></div>
+				<div class="top-meta">11.04.21  /  in Games</a></div>
```

Comment:
remove the </a> as well

Reply:
that's just the date of release of game, we can remove it if you want but i'm afraid of breaking something

Reply:
no I change it. you don't need to do anything